platform/core/README.md

```diff
@@ -1,8 +1,23 @@
-# Rust implementations of polyPod platform core logic
+# PolyPod platform core
```

Comment:
Nit: Now all that's missing is the lower case _p_ in _polyPod_ :D (It's a branding thing)